styles/components/_beta-dialog-content.scss

```diff
@@ -0,0 +1,95 @@
+.beta-dialog-content {
+  display: flex;
+  overlfow-y: auto;
+}
+
+
+
+.promote-syncano-section {
+  width: 40%;
+  padding: 40px 60px;
+}
+
+.syncano-overview-section {
+  width: 60%;
+  padding: 40px 30px;
+  background-color: #0B0F16;
+}
+
+.dialog {
+  position: fixed;
+  top: 50%;
+  bottom: 50%;
+  left: 50%;
+  height: 80%;
+  width: 70%;
+  transform: translate(-50%, -50%);
+  z-index: 9998;
+  background: #fff;
+  overflow-y: auto;
+}
+
+.close-button-icon {
+  font-size: 40px;
+  color: #666666;
+  cursor: pointer;
+  position: fixed;
+  right: 16%;
+  height:40px;
+  width:40px;
+  top: 11%;
+  z-index: 9999;
+}
+
+
+.social-button{
+  border: 10px;
+  box-sizing: border-box;
+  font-family: sans-serif;
+  cursor: pointer;
+  text-decoration: none;
+  outline: none;
+  position: relative;
+  width: 100%;
+  height: 40px;
+  border-radius: 2px;
+  transition: all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;
+  text-align: center;
+}
+
+@media (max-width: 1000px) {
+  .dialog {
+    width: 90%;
+  }
+  .beta-dialog-content {
+    display: flex ;
+    flex-flow: column wrap;
+  }
+  .syncano-overview-section {
+    flex-wrap: wrap;
+    width:100%;
+  }
+  .promote-syncano-section {
+    width: 100%;
+    padding: 40px 10px;
+  }
+  .overview-row {
+    flex-wrap: wrap;
+    display:flex;
+    width:100%;
+    justify-content: center !important;
+    div {
+      width:100% !important;
+      justify-content: center !important;
+      img {
+        margin: 0 auto;
+      }
+      h2, p {
+        text-align: center;
+      }
+    }
+  }
+  .close-button-icon {
+    right: 6%;
+  }
+}
```

Comment:
Please set this: https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842